recipes/pymolpro/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pymolpro" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/molpro/pymolpro/archive/{{ version }}.tar.gz
+  sha256: 1c4fa2540d6e5fbfd0be661348b2b28e967261250daf9ec87b365ca4929bbe5b
+
+build:
+  skip: True  # [win]
```

Comment:
Looks like this one can be noarch. If the reason to skip Windows is a missing dependency it should be fine to make it noarch and "outsource" the problem to the solver at install time.

Reply:
Yes, `no arch` is what I started with, and would be better and simpler, but the windows check then failed because the prerequisite `pysjef` is not ported to windows. I can revisit that to understand why and try to avoid, perhaps.

Reply:
That is fine. The missing dependency won't allow for a Windows installation and, when `pysjef` is ready on Windows, this package won't require any change to work. It is OK, and recommended, to make it noarch in this case.

Reply:
So it's now back to `noarch`, with the consequent failure of the Windows build, and if that can be condoned, it seems to me to be ready now.